lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/authentication/KerberosConfiguration.java

```diff
@@ -99,10 +107,16 @@ public KerberosConfiguration build()
 
             keytabLocation.ifPresent(
```

Comment:
Should we validate here that keytab cannot be used together with cache location?

Reply:
We have added validation as a part of Hadoop and Hive config. Will also add an additional check here

Reply:
Added them.